src/game_frontend/src/pages/Dashboard/layout.tsx

```diff
@@ -11,6 +11,9 @@ const Layout = () => {
       className="exchange"
       style={{
         backgroundImage: 'url("/assets/images/space-exchange-bg.webp")',
+        width: '100vw',           // Full viewport width
+        height: '100vh',         // Full viewport height
```

Comment:
set `min-height : 100vh` and `height : auto`

Reply:
![Screen Recording 2024-12-06 at 4 54 02 PM](https://github.com/user-attachments/assets/a17e26e4-97df-47bd-a00b-3ff43c0cf42b)
 If  follow your suggestion